### DISCUSSION TITLE
bibtex installing problems

### DISCUSSION THREAD

**Discussion by cirocso** [retorquere/zotero-better-bibtex#2103] [Q&A]
Hi, 
I have just downloaded the "Zotero-better-bibtex-6.4.3.xpi" as well as "zotero-better-bibtex-6.4.3.tar" and "zotero-better-bibtex-6.4.3" zip files. They are in the download folder in my computer.
Once I click the xpi file it does not work.
Can someone help me to find out what have  I done wrong?
Thanks,
Ciro #

**Comment by retorquere** [comment]
In Zotero, go to the tools menu, then addons, then under the gear menu choose install from file.

**Reply by cirocso** [reply]
Sorry for my stupidity. It was so clear! It is working yet! Thank you very much.

**Reply by retorquere** [reply]
No worries
